### PR TITLE
Adds slack notifier wrapper to send a message to a specific channel

### PR DESCRIPTION
**Pull request by 0mega**
`#send` method might be improved and just receive arguments to fill a template warning message.
E.g.: `#send(feature_review_link, username, date_time, channel)` after we define what should be included in the message.

It's also possible to direct the message to a specific user, instead of sending to a channel.

### CODE REVIEW

**Comment by 0mega** [comment]
ping @FundingCircle/black-bat

File: .env.development.example [hunk 1]
```diff
@@ -17,3 +17,6 @@ DEPLOY_REGIONS=gb,us
 GITHUB_REPO_STATUS_ACCESS_TOKEN=example_token
 JIRA_HOST_NAME=hostname_of_your_jira_sandbox
 DATA_MAINTENANCE=false
+
+# Slack notifier Web hook
```
Comment (dideler) [inline .env.development.example]: Nitpick: Comment seems redundant, can clarify what it's for in the example value.

File: lib/slack/notifications.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require 'slack-notifier'
+
+class SlackNotifier
+  def initialize
+    @notifier = Slack::Notifier.new ENV['SLACK_WEB_HOOK']
```
Comment (dideler) [inline lib/slack/notifications.rb]: Will sending blow up if the notifier was initialised with `nil`?

**Comment by andistuder** [comment]
LGTM

**Comment by 0mega** [comment]
The content of this PR will be part of #99

**Pull request closed without merging**